AdobeStockImageAdminUi/Test/Mftf/Section/AdobeStockSaveImagePreviewSection.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<sections xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:mftf:Page/etc/SectionObject.xsd">
+    <section name="AdobeStockSaveImagePreviewSection">
+        <element name="name" type="text" selector=".adobe-stock-save-preview-prompt input.admin__control-text[name='name']"/>
+        <element name="mediaImage" type="text" selector="img[alt='{{name}}.jpeg']" parameterized="true"/>
```

Comment:
I believe we should use a more unique selector here. The webdriver will look for `img[alt='{{name}}.jpeg]` on the entire page and there's a chance to spot the same image in the grid or so some day.

Reply:
Agree. Updated the selector.